client/my-sites/theme/main.jsx

```diff
@@ -453,7 +453,10 @@ class ThemeSheet extends React.Component {
 
 	getDefaultOptionLabel = () => {
 		const { defaultOption, isActive, isLoggedIn, isPremium, isPurchased } = this.props;
-		if ( isLoggedIn && ! isActive ) {
+		if ( isActive ) {
+			// Customize size
+			return i18n.translate( 'Customize site' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Customize your site' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/change/theme-switch-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).